### PR TITLE
Allows us to use some new fields

### LINKED ISSUE #2: nmcli nm permissions command missing from the NM Command list
Attempting the command causes this error:

         nmcli.nm.permissions()
         Traceback (most recent call last):
         File "<stdin>", line 1, in <module>
         AttributeError: 'NMCommand' object has no attribute 'permissions'

Output of the command when run from the command line:

> PERMISSION                                                VALUE     
org.freedesktop.NetworkManager.enable-disable-network     yes       
org.freedesktop.NetworkManager.enable-disable-wifi        yes       
org.freedesktop.NetworkManager.enable-disable-wwan        yes       
org.freedesktop.NetworkManager.enable-disable-wimax       yes       
org.freedesktop.NetworkManager.sleep-wake                 no        
org.freedesktop.NetworkManager.network-control            yes

### PR DESCRIPTION
**Pull request by cjds**
Fixes #2 #1

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,9 +5,23 @@ Python wrapper exposing nmcli as an API
...
+
+> pip install python-nmcli
+
 Example
 =======
 
+Example code that uses this library
+ 
+    import nmcli
+    print nmcli.nm.permissions()
```
Comment (ZachGoldberg) [inline README.rst:19]: Should't this be

`from nmcli import nm`
`print nm.permissions()`
`...`

**Review by ZachGoldberg — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (cjds) [reply README.rst:19]: Both work. I think it's just a style thing

Comment (ZachGoldberg) [reply README.rst:19]: I'd opt to at least be consistent in the README if possible

File: nmcli/shell.py [hunk 2]
```diff
@@ -64,13 +66,16 @@ def nmcli(obj, command=None, fields=None, multiline=False):
...
+    if command:
+        if ("%s %s" % (obj,command)) in NMCLI_FIELDS:
+            fields=NMCLI_FIELDS[("%s %s" % (obj,command))]
+
     args = ['nmcli', '--terse', '--fields', ",".join(fields), obj]
 
     if command:
         args += shlex.split(command)
-
+    print args
```
Comment (ZachGoldberg) [inline nmcli/shell.py:77]: Remove

**Comment by cjds** [comment]
Can you merge this in and re-release on pip, so this functionality can be pip installed?

**Comment by ZachGoldberg** [comment]
There are some changes that I'd like to see before merging... I'll make them then push & merge now

**Comment by cjds** [comment]
Sounds good to me. If you would like me to make them, just tell me what to do

**Pull request merged**